pkg/cable/ipsec/strongswan.go

```diff
@@ -203,6 +204,8 @@ func (i *strongSwan) ConnectToEndpoint(endpoint types.SubmarinerEndpoint) (strin
 		return "", fmt.Errorf("error loading connection %q with config %#v into charon: %v", endpoint.Spec.CableName, ikeConf, err)
 	}
 
+	i.remoteEndpoints[endpoint.Spec.CableName] = endpoint.Spec
```

Comment:
I guess we're safe to assume thread safety here - we know the lone caller locks.

Reply:
Yes, that's the assumption, we could add a lock to the cable methods if it changed.